contracts/core/liquidators/impl/TwoAssetPriceBoundedLinearAuction.sol

```diff
@@ -83,7 +82,7 @@ contract TwoAssetPriceBoundedLinearAuction is LinearAuction {
 
```

Comment:
In the Javadocs of the constructor, please note that the `rangeStart` and `rangeEnd` need to be 18 decimal values.

In addition, consider adding a require that it has to be less than 100% and a multiple of a basis point